app/src/main/java/no/agens/androidtweaks/MyTweaks.java

```diff
@@ -1,19 +1,31 @@
 package no.agens.androidtweaks;
 
 
+import android.util.Log;
+
 import java.util.ArrayList;
 import java.util.List;
 
 import no.agens.androidtweakslibrary.models.Tweak;
 import no.agens.androidtweakslibrary.models.TweakBoolean;
+import no.agens.androidtweakslibrary.models.TweakClosure;
+import no.agens.androidtweakslibrary.models.TweakStore;
 
 public class MyTweaks {
     public static final TweakBoolean uppercase = new TweakBoolean("Styling", "Fonts", "Uppercase", false);
     public static final TweakBoolean greenButton = new TweakBoolean("Styling", "Colors", "Green", false);
-
+    public static final TweakBoolean useEmojis = new TweakBoolean("User", "User data", "Use emoji instead of photo", false);
+    public static final TweakClosure deleteUserData = new TweakClosure("User", "User data", "Delete user data", new TweakStore.Callback() {
```

Comment:
Should we pass the context as a parameter? We can get the context from the TweakStore.

@Ehyeh-Asher-Ehyeh, what do you think?

Reply:
I think we shouldn't; in general, holding on to Context in a singleton is an anti-pattern.